app/src/main/java/com/pixlee/pixleeandroidsdk/ui/gallery/GalleryFragment.java

```diff
@@ -248,19 +289,21 @@ public void onError(String error) {
      */
     @Override
     public void onComplete(ArrayList<PXLPhoto> photos) {
+        setLoading(false);
         if (photos == null) {
             return;
         }
-        album.openedWidget(PXLWidgetType.horizontal);
-        album.openedWidget("gallery");
-        album.loadMore();
+        //album.openedWidget(PXLWidgetType.horizontal);
```

Comment:
there are a lot of commented code in this PR in general. can we have explanations for the commented code or just deletion of the commented code? theres no point in just commenting out code

Reply:
Agree! I'll add GUI views for developers to try these features.

Reply:
I implemented all sort options and half of the filter options for the SDK users to understand about it and left more explanations for the other half of the filter options.